cardinal/server/handler/health.go

```diff
@@ -11,10 +11,10 @@ type GetHealthResponse struct {
 
 // GetHealth godoc
 //
-//	@Summary		Get information on status of world-engine
-//	@Description	Displays information on http server and world game loop
+//	@Summary		Get the World Engine's health status
```

Comment:
We seem to be leaning towards "cardinal" instead of "world engine". I think this should be "Get Cardinal's health status".